Jenkinsfile

```diff
@@ -74,15 +74,24 @@ pipeline
 							expression { map_deb_distro[env.DISTRO] != null }
 							expression { map_rpm_distro[env.DISTRO] != null }
 						} }
-						steps { publishPackage(artifactoryRoot, map_deb_distro[env.DISTRO], map_rpm_distro[env.DISTRO]) }
+						steps
+						{
+							lock(label: 'elastio-vmx', quantity: 1, resource : null)
+							{
+								publishPackage(artifactoryRoot, map_deb_distro[env.DISTRO], map_rpm_distro[env.DISTRO])
```

Comment:
Do we use a lot of resources during publishing packages?

Reply:
Using compressing for folders - CPU usage..